walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,72 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 
+/**
+ * Finds time ranges when meeting could happen in a day assuming given events will be sorted from
+ * beginning of the day till the end.
+ */
 public final class FindMeetingQuery {
```

Comment:
Is there a test for this also?

Reply:
Do you mean testing if events are sorted?